about.html

```diff
@@ -12,62 +13,110 @@
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
-      <footer>
-        <nav>
-          <a href="index.html">Home</a>
-          <a href="#">About</a>
-          <a href="#">Products</a>
-          <a href="#">Blog</a>
-          <a href="#">Contact</a>
+  <div class="container about-page">
+
+    <header>
+      <section class="nav-content">
+        <img src="img/lambda-black.png" alt="Lambda Logo">
+
+        <nav class="nav-bar">
+          <a href="index.html" class="nav-item">Home</a>
+          <a href="about.html" class="nav-item">About</a>
+          <a href="#" class="nav-item">Products</a>
+          <a href="#" class="nav-item">Blog</a>
+          <a href="#" class="nav-item">Contact</a>
         </nav>
-      </footer>
-    </div><!-- container -->        
+      </section>
+
+      <section class="top-content">
+        <img src="img/jumbo-about.png" class="jumbo-image">
+
+        <div class="text-container text-container-about">
+          <h2>About</h2>
+          <p>
+            Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to
+            corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow
+            the holistic world view of disruptive innovation via workplace diversity and empowerment.
+          </p>
+        </div>
+      </section>
+
+    </header>
+
+    <section class="middle-content middle-content-about">
+      <div class="boxes about-boxes">
```

Comment:
This div might not be necessary - A section is essentially a more semantic div, and you don't have any other children in your section tag that would require grouping everything in the div